TestAssets/TestPackages/dotnet-portable/Program.cs

```diff
@@ -10,6 +12,12 @@ public class Program
         public static void Main(string[] args)
         {
             Console.WriteLine("Hello Portable World!");
+
+            var coreAssembly = typeof(object).GetTypeInfo().Assembly;
+            string coreFolder = Path.GetDirectoryName(coreAssembly.Location);
+            string frameworkVersion = Path.GetFileName(coreFolder);
```

Comment:
I saw that logic, but I didn't know the reason behind it and the code seemed uglier.  Since there's a reason behind it, I'll update it to use `FX_DEPS_FILE`.